From 6f765031992459594f35ffe65cbf2a735b3dd09e Mon Sep 17 00:00:00 2001
From: Philip Balister <philip@opensdr.com>
Date: Sun, 11 Oct 2026 13:08:08 -0400
Subject: [PATCH] CMakeLists.txt: Don't require the Boost system component

Boost.System has been header-only since 1.69, and Boost 1.89 stopped
shipping the boost_system stub library and its CMake config, so
find_package(Boost ... system) fails. Nothing links against it.

Upstream-Status: Pending
Signed-off-by: Philip Balister <philip@opensdr.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3a6b0da..d2adbcb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -119,7 +119,7 @@ endif()
 ########################################################################
 # Find boost
 ########################################################################
-find_package(Boost "1.65" REQUIRED chrono thread system)
+find_package(Boost "1.65" REQUIRED chrono thread)
 
 if(NOT Boost_FOUND)
     message(FATAL_ERROR "Boost required to compile osmosdr")
-- 
2.56.0

